view/frontend/web/js/price-utils.js

```diff
@@ -26,6 +26,27 @@ define([
         return (new Array(times + 1)).join(string);
     }
 
+    function formatPriceLocale(amount, format, isShowSign)
+    {
+        var s = '',
+            precision, pattern, locale, r;
+
+        format = _.extend(globalPriceFormat, format);
+        precision = isNaN(format.requiredPrecision = Math.abs(format.requiredPrecision)) ? 2 : format.requiredPrecision;
+        pattern = format.pattern || '%s';
+        locale = window.LOCALE || 'en-US';
+        if (isShowSign === undefined || isShowSign === true) {
+            s = amount < 0 ? '-' : isShowSign ? '+' : '';
+        } else if (isShowSign === false) {
+            s = '';
+        }
```

Comment:
s = '';
        if (isShowSign === undefined || isShowSign === true) {
            s = amount < 0 ? '-' : isShowSign ? '+' : '';
        }

Reply:
Please, try this and I'll merge your code. Thanks all for yours implication in order to make better this module. @npluimers @pfortin-expertime @CptCharlesG @verhaeghep